test/image_test.py

```diff
@@ -112,7 +130,7 @@ def test_image_requirements_txt(servicer, client):
         assert any(b"banana" in f.data for f in layers[0].context_files)
 
 
-def test_empty_install(servicer, client):
+def test_empty_install(builder_version, servicer, client):
```

Comment:
Could also make `builder_version` an `autouse` fixture if it doesn't need to be referenced in most tests

Reply:
There were a couple of tests that I didn't want it to run for (i.e. the ones testing the version-based exceptions), but yeah it was pretty annoying adding it everywhere and maybe it's fine running there since I am mocking it anyway in those tests 🤔

Reply:
Tried refactoring the versioned methods into a class and using a class scoped fixture, but that was incompatible with some of the other fixtures that we need to have running so I gave up.